### PR TITLE
Add volume info to machine image response (#1011)

### PR DESCRIPTION
**Pull request by eyaroslavtsev**
Ref
https://github.com/TheWeatherCompany/grid-api/pull/1018

### CODE REVIEW

File: src/main/java/org/dasein/cloud/compute/SnapshotVolume.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package org.dasein.cloud.compute;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * @author Eugene Yaroslavtsev
+ * @since 18.07.2014
+ */
+public class SnapshotVolume {
```
Comment (clstokes) [inline src/main/java/org/dasein/cloud/compute/SnapshotVolume.java]: Please rename this class to `MachineImageVolume`.

File: src/main/java/org/dasein/cloud/compute/MachineImage.java [hunk 2]
```diff
@@ -230,6 +231,7 @@
     private Map<String,String> tags;
     private MachineImageType   type;
     private VisibleScope       visibleScope;
+    private Collection<SnapshotVolume> snapshotVolume;
```
Comment (clstokes) [inline src/main/java/org/dasein/cloud/compute/MachineImage.java]: Rename this collection to `volumes`

**Comment by clstokes** [comment]
@eyaroslavtsev please see my in-line code comments.

**Comment by eyaroslavtsev** [comment]
@clstokes class and field was renamed. please, check it

**Comment by clstokes** [comment]
@eyaroslavtsev we need changes in https://github.com/TheWeatherCompany/dasein-cloud-aws/pull/119 and https://github.com/TheWeatherCompany/grid-api/pull/1018 for this right?

**Comment by clstokes** [comment]
I see you already made the changes in dasein-cloud-aws. I made the changes in grid-api.

**Comment by eyaroslavtsev** [comment]
@clstokes Unfortunately PR was not ready. 
I apologize for change for _grid-api_ did not come.
Also confusion with VolumeType#valueOf / VolumeType#valueOfType

**Pull request merged**